frontend/src/app/containers/ExperimentPage/types.js

```diff
@@ -4,6 +4,49 @@ import type {
   SendToGAProps
 } from '../../containers/types';
 
+export type ExperimentPropsFromApp = {
```

Comment:
These types could use some further reorg & DRYing up. But, I erred on the side of verbosity to make sure I captured everything.